src/config.rs

```diff
@@ -465,7 +465,7 @@ make_config! {
         emergency_access_allowed:    bool,   true,   def,    true;
         /// Password iterations |> Number of server-side passwords hashing iterations.
         /// The changes only apply when a user changes their password. Not recommended to lower the value
```

Comment:
This comment is not valid anymore
```suggestion
        /// The changes apply when a user logs in. The minimum allowed value is 100000.
```

Reply:
Actually, that comment was never valid.

Reply:
Thanks, i have changed this at the `config.rs` and `.env.template` file.